test/e2e/specs/test.spec.js

```diff
@@ -19,15 +19,15 @@ describe('Wallet App Test Cases', { execTimeout: DEFAULT_EXEC_TIMEOUT }, () => {
       });
     });
     it(`should open the web wallet URL successfully`, () => {
-      cy.visit(`${userConfig.webwalletURL}`);
+      cy.visit('/');
```

Comment:
Shouldn't we do it conditionally wheb AGORIC_NET is local?

Reply:
All networks should use `localhost:3000` for testing. Previously, all networks used `localhost:3000` except when `AGORIC_NET` was set to 'local,' in which case `https://wallet.agoric.app/` was used. You can see this in the `webWalletURL` parameter [here](https://github.com/Agoric/wallet-app/blob/main/test/e2e/test.utils.js#L55C1-L71C3).